build_scripts/windows/scripts/build.cmd

```diff
@@ -121,21 +124,42 @@ if not exist %PYTHON_DIR% (
     del get-pip.py
     echo Pip set up successful
 
-    dir .
+    REM setuptools is not installed by default in Python 3.12, but it is required by some dependencys
+    REM See https://github.com/Azure/azure-cli/pull/27196
+    echo Installing setuptools
+    %PYTHON_DIR%\python.exe -m pip install setuptools
```

Comment:
Although `setuptools` is added to `azure-cli `dependency list in https://github.com/Azure/azure-cli/pull/27196, `azure-cli` is installed with `--no-deps`. Need to install it manually.

In 3.11, `wheel` is also installed when running `get-pip.py`. However, `wheel` is not included in final package in Linux and macOS.

Ref:
`setuptools` remove `wheel` dependency: https://github.com/pypa/setuptools/issues/1386
`pip` remove `wheel` dependency: https://github.com/conda-forge/pip-feedstock/issues/124

```
Installing pip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2213k  100 2213k    0     0  47.2M      0 --:--:-- --:--:-- --:--:-- 50.2M
Collecting pip
  Downloading pip-24.2-py3-none-any.whl.metadata (3.6 kB)
Collecting setuptools
  Downloading setuptools-75.1.0-py3-none-any.whl.metadata (6.9 kB)
Collecting wheel
  Downloading wheel-0.44.0-py3-none-any.whl.metadata (2.3 kB)
Downloading pip-24.2-py3-none-any.whl (1.8 MB)
   ---------------------------------------- 1.8/1.8 MB 33.0 MB/s eta 0:00:00
Downloading setuptools-75.1.0-py3-none-any.whl (1.2 MB)
   ---------------------------------------- 1.2/1.2 MB 30.8 MB/s eta 0:00:00
Downloading wheel-0.44.0-py3-none-any.whl (67 kB)
Installing collected packages: wheel, setuptools, pip
  WARNING: The script wheel.exe is installed in 'D:\a\_work\1\s\build_scripts\windows\artifacts\Python\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The scripts pip.exe, pip3.11.exe and pip3.exe are installed in 'D:\a\_work\1\s\build_scripts\windows\artifacts\Python\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed pip-24.2 setuptools-75.1.0 wheel-0.44.0
Pip set up successful
```